anonymous.js

```diff
@@ -0,0 +1,12 @@
+// const array1 = [2,3,4,5];
+// const square= array1.map(x=> x*x );
+// console.log(square);
+
+
+// var a = 10;
+// var b = 20;
+// (() => {return a + b})(a ,b)
```

Comment:
```suggestion
// (() => a + b)()
```
Posto su a i b globalne varijable, ne moramo nista da prosledimo funkciji + imamo samo return, tako da nam ne trebaju {} zagrade